subprojects/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/impldeps/BaseGradleImplDepsIntegrationTest.groovy

```diff
@@ -48,24 +48,6 @@ abstract class BaseGradleImplDepsIntegrationTest extends AbstractIntegrationSpec
         """
     }
 
-    static String junitDependency() {
-        """
-            dependencies {
-                testImplementation 'junit:junit:4.13.1'
-            }
-        """
-    }
-
-    static String spockDependency() {
-        """
-            dependencies {
-                testImplementation('org.spockframework:spock-core:2.1-groovy-3.0') {
-                    exclude group: 'org.codehaus.groovy'
```

Comment:
Do we care about this exclude now or is there not a conflict between Spock and Gradle's Groovy?

Reply:
Hmm, I'm not sure. I'm tempted to try without it and see if JDK 19 passes the tests, if it does then there's no issues.